java-sdk-logging/java-sdk-logging-bom/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.api</groupId>
+    <artifactId>java-sdk-logging-parent</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <packaging>pom</packaging>
+
+  <artifactId>java-sdk-logging-bom</artifactId>
```

Comment:
Wonder if better be more specific like `java-sdk-logging-extensions-bom`? 
`java-sdk-logging-bom` might be a bit confusing as it sounds like containing all logging component, and I believe since we do not publish this regularly, this is not intended to host the core client debug logging logics.

Reply:
Done.